lms/djangoapps/completion/management/commands/migrate_contactid.py

```diff
@@ -0,0 +1,86 @@
+"""
```

Comment:
This command doesn't have any relationship with completion, you'd better move  to another django app

Reply:
What Djangoapp? @andrey-canon